tensorflow/contrib/ignite/BUILD

```diff
@@ -31,39 +32,88 @@ tf_custom_op_library(
     deps = [":dataset_kernels"],
 )
 
+tf_custom_op_library(
```

Comment:
Since `":igfs_kernels"` and `":dataset_kernels"` share some symbols (in `":ignite_client"`), to avoid the ODR issues, it'll be easiest to merge "_dataset_ops.so" and "_igfs_ops.so" into a single shared object (and hence `tf_custom_op_library()` rule).

Reply:
Okay, good point. I've merged them into a single shared library `_ignite_ops.so`.